README.md

```diff
@@ -1,110 +1,124 @@
 # The missing introduction to APCA
 
-## What is APCA?
+## _What is APCA in three sentences:_
+It is a supra threshold visual contrast prediction model that is specifically tuned to readability on self-illuminated displays. It follows human perception, including critical aspects such as spatial frequency, meaning font weight and size. The APCA guidelines also inform various use cases, such as body text, fluent text, spot reading text, and non-text items.
+
+### _WCAG 2 compatibility?_
+APCA is enough of a paradigm shift that it is not backwards compatible with the old WCAG&nbsp;2 contrast, and WCAG&nbsp;3 is some time away from being the recommendation.
+
+So in the meantime, there is the Bridge-PCA, which is backward compatible to WCAG&nbsp;2, but using APCA technology. One of the urgent reasons Bridge-PCA is needed is the rise in popularity of _"Dark Mode."_
+
+### _Bridge PCA for Dark Mode_
+As shown above, WCAG&nbsp;2 contrast fails most with dark colors, and the results are relatively meaningless for colors for Dark Mode. APCA and Bridge&nbsp;PCA do very well with contrast prediction and guidance for dark mode.
+
+### Here's a [short linktree](https://linktr.ee/Myndex) of related links.
+### And a [more in depth](https://git.myndex.com) link catalog.
```

Comment:
Headings must be used to start a section, not to highlight some text. See [WCAG 2.1 2.4.6](https://www.w3.org/TR/WCAG21/#headings-and-labels).